crescent/commands/options.py

```diff
@@ -140,191 +127,191 @@ def __get__(self, inst: Any | None, cls: Any) -> Any:
 
 
 DEFAULT = TypeVar("DEFAULT")
-USER = TypeVar("USER", bound=Type[User])
-ROLE = TypeVar("ROLE", bound=Type[Role])
+USER = TypeVar("USER", bound=type[User])
+ROLE = TypeVar("ROLE", bound=type[Role])
```

Comment:
This feels more consistent then using `typing.Type` to me
```suggestion
USER = TypeVar("USER", bound="type[User]")
ROLE = TypeVar("ROLE", bound="type[Role]")
```